nusantara/nusa_datasets/xl_sum/xl_sum.py

```diff
@@ -0,0 +1,152 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+from nusantara.utils import schemas
+import jsonlines
+from nltk.tokenize.treebank import TreebankWordDetokenizer
+
+_CITATION = """\
+@inproceedings{hasan2021xl,
+  title={XL-Sum: Large-Scale Multilingual Abstractive Summarization for 44 Languages},
+  author={Hasan, Tahmid and Bhattacharjee, Abhik and Islam, Md Saiful and Mubasshir, Kazi and Li, Yuan-Fang and Kang, Yong-Bin and Rahman, M Sohel and Shahriyar, Rifat},
+  booktitle={Findings of the Association for Computational Linguistics: ACL-IJCNLP 2021},
+  pages={4693--4703},
+  year={2021}
+}
+"""
+
+_DATASETNAME = "xl_sum"
+
+_DESCRIPTION = """\
+XL-Sum is a large-scale multilingual summarization dataset that covers 45 languages including Indonesian text summarization.
+The dataset is based on article-summary pairs from BBC, is highly abstractive, concise, and of high quality, as indicated by human and intrinsic evaluation.
+"""
+
+_HOMEPAGE = "https://github.com/csebuetnlp/xl-sum"
+
+_LICENSE = "CC-BY-NC-SA 4.0"
+
+_URLS = {
+    _DATASETNAME: "https://huggingface.co/datasets/csebuetnlp/xlsum/resolve/main/data/indonesian_XLSum_v2.0.tar.bz2",
+}
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION]
+
+_SOURCE_VERSION = "2.0.0"
```

Comment:
Is 2.0.0 ok @SamuelCahyawijaya ? should it be 1.0.0 maybe?

Reply:
Yes, the dataset has two versions. https://github.com/csebuetnlp/xl-sum Here we use the new version

Reply:
This one is correct I guess since the original dataset is version 2.0